.pkg/linux/install

```diff
@@ -38,19 +38,19 @@ BI_MGT="$ITL$B_MGT"
 ####
 
 # AppImage
-app_tag=nym-vpn-app-v1.3.0-nectarine.2
-app_version=1.3.0-nectarine.2
+app_tag=nym-vpn-app-v1.3.0
+app_version=1.3.0
 appimage_url="https://github.com/nymtech/nym-vpn-client/releases/download/$app_tag/NymVPN_${app_version}_x64.AppImage"
 app_deb_url="https://github.com/nymtech/nym-vpn-client/releases/download/$app_tag/nym-vpn-app_${app_version}_amd64.deb"
 desktop_url="https://raw.githubusercontent.com/nymtech/nym-vpn-client/fb526935/nym-vpn-app/.pkg/app.desktop"
 icon_url="https://raw.githubusercontent.com/nymtech/nym-vpn-client/fb526935/nym-vpn-app/.pkg/icon.svg"
 
 # nym-vpnd
-vpnd_tag=nym-vpn-core-v1.3.0-beta.1
+vpnd_tag=nym-vpn-core-v1.3.1
 # ⚠ there are inconsistencies in the package version naming…
-vpnd_version=1.3.0-beta.1
-vpnd_deb_ver=1.3.0~beta.1
-vpnd_deb_ver_url=1.3.0.beta.1
+vpnd_version=1.3.1
```

Comment:
Get the version from the tag? Or other way around?

Reply:
This is an internal custom script, we're using to get quickly setup, especially for testing.
I need to swap the versions frequently, depending on what current build is the current hotspot, dev nightly beta or latest stable.
So automation is not really possible.

Reply:
Imo, the issue is everything in this repo is push protected and require PR + 3d party approval.
Such change, like this script update, does not require that process.